ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:   3.0
 name:            ouroboros-consensus-cardano
-version:         0.6.1.0
+version:         0.6.2.0
```

Comment:
Same with versions, I think they are modified right before the release, but I am not 100% confident on this one.

Reply:
Got it! Thanks!